src/rgw/rgw_auth_s3.h

```diff
@@ -103,6 +103,7 @@ class AWSAuthStrategy : public rgw::auth::Strategy,
   S3AnonymousEngine anonymous_engine;
   ExternalAuthStrategy external_engines;
   LocalEngine local_engine;
+  OPAEngine opa_engine;
```

Comment:
ah, I get it now.  since OPAEngine turned out to be a dead end--my fault!--this commit is introducing something only for later commits in the initial PR to discard;  I think maybe we should rebase this history away unless it leads somewhere?